docs/quickstart.rst

```diff
@@ -292,6 +296,7 @@ We can now train our final model and visualize it with the ``PredictionError`` v
 
 .. plot::
     :include-source: False
+    :context: close-figs
```

Comment:
Thanks for adding these too - I was a bit confused because I had added these directives to the quickstart, but they're still in the unmerged #729. I think we have some things to figure out there still, so it's good to get them in early and it should merge just fine (probably).

Reply:
Ah, I'm sorry @bbengfort, I wasn't tracking that PR closely and thought it was just pcoords. Shall I revert this change so that we don't risk a merge conflict?

Reply:
No, I think this will be merged sooner than #729 (sorry @lwgray) and would rather have these options in than out. I can handle the merge conflict if it comes up - it won't be too difficult to sort in this case.